Automattic/woocommerce-payments
# Show a deposit schedule notice on the deposits list page

Fixes #7877

Based on PR from #7875 since it relies on changes to the `DepositSchedule` component.

#### Changes proposed in this Pull Request

Adds a notice to the Deposits List screen to communicate the deposit schedule for the account. The "next deposit" part of the message is **not** rendered here since this screen can include multiple currencies with differing next deposit dates.

This notice will only be rendered if deposits are unrestricted (not blocked or on a restricted schedule).

This notice is dismissable by the user. When this occurs, the dismissal state will persist until the user changes their deposit schedule settings.

https://github.com/Automattic/woocommerce-payments/assets/3147296/4d8e62ef-724e-41b8-b33b-f3b020594be5





#### Testing instructions





* Check that the notice is rendered on the Deposits List screen for accounts without deposit restrictions (use MC tools to add/remove restrictions to account)
* Check that the message is not rendered if
	* Deposits blocked
	* Deposits restricted
* Dismiss the notice and refresh the page. Ensure the notice remains hidden.
* Change your deposit schedule via Payments → Settings.
* Ensure the notice is now visible on the Deposits List screen and shows the correct schedule.

-------------------


**Post merge**
